### PR TITLE
BACKEND: Adds 18+ pop for each sex and race group, for use in AHR `pct_share` calcs

### PR DESCRIPTION
**Pull request by benhammondmusic**
## Description

- [x] creates a second census population estimates datasource
- [x] processes the csv files from census by selecting unique combos of state/group/year and sums populations for 18+ only
- [x] writes to new BQ tables: `sex_age` and `race_age` for `state` and `national`

## Motivation and Context

#1786 

## How has this been tested?

- Adds automated tests 

Manually compared the results a few different ways
- using Numbers I cropped out all rows that weren't applicable to MALES 18+ in ALABAMA of ANY RACE, specifically HISPANIC or NON-HISPANIC and summing the 2021 column was exactly the same as my output file's corresponding values
Screen Shot 2022-11-03 at 3 22 31 PM

- using a Census file that only has 18+ populations by state (not broken down by race) I compared the state 18+ totals to my calculated race/eth == "ALL". The numbers were in the right ballpark, but this comparison census file is from 2019 and my file calculates 2020 and 2021 values. 
Screen Shot 2022-11-03 at 3 28 48 PM

Screenshot from my BQ showing 4 tables with correct columns
Screen Shot 2022-11-08 at 5 15 02 PM

## Types of changes

- New feature (non-breaking change which adds functionality or content)

### CODE REVIEW

**Review by eriwarr — APPROVED** [review]
Looks good 👍🏿

File: python/datasources/census_pop_estimates_sc.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
+from ingestion.standardized_columns import Race
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util
+import ingestion.standardized_columns as s
```
Comment (joshzarrabi) [inline python/datasources/census_pop_estimates_sc.py:4]: we import this as `std_col` everywhere else, can we keep that pattern for now

File: python/tests/datasources/test_census_pop_estimates_sc.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    assert mock_csv.call_count == 1
+
+    # 4 = 2 demographic breakdowns X 2 geographic breakdowns
+    assert mock_bq.call_count == 4
+    assert mock_gen.call_count == 4
+
+# TEST INNER FUNCTION - RACE BY STATE
+
+
+def test18PlusByRace():
```
Comment (joshzarrabi) [inline python/tests/datasources/test_census_pop_estimates_sc.py:68]: Can you specify these are by state in the function title.

**Review by joshzarrabi — APPROVED** [review]
Looks great, thank you ben

File: python/datasources/census_pop_estimates_sc.py (hunk 1 above)
Comment (benhammondmusic) [reply python/datasources/census_pop_estimates_sc.py]: sure. I got tired of typing lol

File: python/tests/datasources/test_census_pop_estimates_sc.py (hunk 2 above)
Comment (benhammondmusic) [reply python/tests/datasources/test_census_pop_estimates_sc.py]: good call

**Pull request merged**